app/views/devise/passwords/edit.html.erb

```diff
@@ -4,8 +4,14 @@
 
 <h1 class="govuk-heading-l">Change your password</h1>
 
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+<%= form_for(
+  resource,
+  as: resource_name,
+  url: password_path(resource_name),
+  html: { method: :put },
```

Comment:
I know you didn't write this, but isn't it just `method: :put` rather than `html: { method: :put }` 🤔